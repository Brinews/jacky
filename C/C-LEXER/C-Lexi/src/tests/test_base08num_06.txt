
  123456700000000000000000000000x08
