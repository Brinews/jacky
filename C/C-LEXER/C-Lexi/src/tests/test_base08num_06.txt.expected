{123456700000000000000000000000x08 , BASE08NUM , 2}
{ , END_OF_FILE , 4}
